ui/tests/acceptance/access/identity/entities/index-test.js

```diff
@@ -70,22 +72,62 @@ module('Acceptance | /access/identity/entities', function (hooks) {
   });
 
   test('it renders popup menu for external groups with alias', async function (assert) {
```

Comment:
this test seems to be asserting the logic here in this block:
https://github.com/hashicorp/vault/blob/df843a859a1bfe11b2eec554cd1bf390362da974/ui/app/templates/vault/cluster/access/identity/index.hbs#L54-L60

so opted to stub mirage instead of creating the actual external group